Lib/test/test_monitoring.py

```diff
@@ -2003,10 +2004,11 @@ def callback(code, instruction_offset):
         sys.monitoring.set_events(0, 0)
 
 
+@unittest.skipUnless(hasattr(_testinternalcapi, "new_counter_optimizer"),
+                     "Requires counter optimizer")
```

Comment:
`_testinternalcapi` never has `new_counter_optimizer` now, so this test will never run.

The good news is that this test works fine with any optimizer... or none at all! So just remove the decorator, as well as the `setUp` and `tearDown` methods.